drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -689,9 +1028,14 @@ static struct ieee802154_radio_api kw41z_radio_api = {
 	.tx			= kw41z_tx,
 };
 
-NET_DEVICE_INIT(kw41z, CONFIG_IEEE802154_KW41Z_DRV_NAME,
-		kw41z_init, &kw41z_context_data, NULL,
-		CONFIG_IEEE802154_KW41Z_INIT_PRIO,
-		&kw41z_radio_api, IEEE802154_L2,
-		NET_L2_GET_CTX_TYPE(IEEE802154_L2),
-		KW41Z_PSDU_LENGTH);
+NET_DEVICE_INIT(
+	kw41z,                              /* Device Name */
+	CONFIG_IEEE802154_KW41Z_DRV_NAME,   /* Driver Name */
+	kw41z_init,                         /* Initialization Function */
+	&kw41z_context_data,                /* Context data */
+	NULL,                               /* Configuration info */
+	CONFIG_IEEE802154_KW41Z_INIT_PRIO,  /* Initial priority */
+	&kw41z_radio_api,                   /* API interface functions */
+	IEEE802154_L2,                      /* L2 */
+	NET_L2_GET_CTX_TYPE(IEEE802154_L2), /* L2 context type */
+	KW41Z_PSDU_LENGTH);                 /* MTU size */
```

Comment:
I don't see much point for this change. If one wants to know what are the parameters for NET_DEVICE_INIT, looking at net_if.h would solve that.

Reply:
I find it useful.